io/plugins/eu.esdihumboldt.hale.io.json/src/eu/esdihumboldt/hale/io/json/internal/JsonInstanceConstants.java

```diff
@@ -37,4 +39,23 @@ public interface InstanceJsonConstants {
 	 */
 	public static final String PREFIX_INSTANCE_JSON = "hj";
 
+	/**
```

Comment:
These are constants related to the writer implementation and not to the generic Instance to Json or Json to Instance conversion, so it would be preferrable to not have them added here.